sys/drv/bcm2835_emmc.c

```diff
@@ -28,8 +28,10 @@ typedef struct bcmemmc_state {
   spin_t lock;           /* Covers `pending`, `intr_recv` and `emmc`. */
   uint64_t rca;          /* Relative Card Address */
   uint64_t host_version; /* Host specification version */
-  volatile uint32_t pending; /* All interrupts received */
-  emmc_error_t errors;
+  volatile uint32_t pending;  /* All interrupts received */
+  emmc_error_t errors;        /* Error flags */
+  emmc_error_t ignore_errors; /* Error flags that do not cause invalidation of
```

Comment:
`ignore_errors` sounds like a flag that (globally) disables error handling. Maybe `ignored_errors`?

Reply:
Good suggestion. I changed the field name to `ignored_errors`.